armsrc/hitagS.c

```diff
@@ -1271,7 +1253,7 @@ static int selectHitagS(const lf_hitag_data_t *packet, uint8_t *tx, size_t sizeo
 
     //select uid
     txlen = 0;
-    cmd = 0x00;
+    cmd = 0x00; // 00000 SELECT UID
```

Comment:
don't we have DEFINE's  for the commands in protocol.h ?

Reply:
It seems that hitag2/S commands are missing in protocol.h. Should I create another PR to move all commands define to protocol.h?

Reply:
yeah,  that would be great